### ISSUE TITLE
Plugin seems confused b/w F and C

### ISSUE THREAD

**Issue by danberger** [iklein99/homebridge-smartthings#247]
**Describe The Bug:**
I am using this plugin to connect two ACs to Homekit. Whenever I load the plugin, I get an error and the dial’s digits seem to be converting F to C.

I have tried switching F and C on both the AC unit and in the AC widget in HomeKit.

**Logs:**

```
1/15/2024, 1:02:40 PM SmartThings Ik BridgeERROR[SmartThings Ik Bridge 5B3C@Air Conditioner@@Target Temperature] characteristic was supplied illegal value: number 61 exceeded maximum of 38
1/15/2024, 1:02:40 PM SmartThings Ik BridgeERROR[SmartThings Ik Bridge 5B3C@Air Conditioner@@Target Temperature] characteristic was supplied illegal value: number 61 exceeded maximum of 38
```

**Plugin Config:**

```json
{
    "platform": "HomeBridgeSmartThings",
    "name": "Smartthings Plug (IK)",
    "BaseURL": "https://api.smartthings.com/v1/",
    "GarageDoorMaxPoll": 40,
    "AccessToken": "a373cfb4-4566-4ee4-aa1d-ae1f73042f9b"
}
```

**Screenshots:**
![IMG_9494](https://github.com/iklein99/homebridge-smartthings/assets/613302/5d51d7ef-711d-4e5f-a8e5-6854238bda1b)

**Environment:**

* **Plugin Version**: 1.5.20
* **Homebridge Version**: 1.6.0
* **Node.js Version**: 16.20.2
* **Operating System**: 5.15.84-sunxi

**Comment by josephmarino1** [comment]
Experiencing the same bug here.

**Comment by iklein99** [comment]
Just published 1.5.21 to fix this.  I really can't test as I don't have an AC.  Please give it a shot and let me know if it solved the issue.

**Comment by danberger** [comment]
> Just published 1.5.21 to fix this. I really can't test as I don't have an AC. Please give it a shot and let me know if it solved the issue.

NICE! It works man. Thank you!!!!
